mutiny-core/src/onchain.rs

```diff
@@ -349,9 +346,20 @@ impl OnChainWallet {
         log_debug!(self.logger, "Unsigned PSBT: {psbt}");
         let finalized = wallet.sign(&mut psbt, SignOptions::default())?;
         log_debug!(self.logger, "finalized: {finalized}");
-        self.label_psbt(&psbt, labels)?;
```

Comment:
I see why it should change for estimating but why for `spend_utxos_to_output`? Shouldn't that behave just like `sweep` would?

Reply:
Just moved the labeling logic for fee estimation, no other functionality changes.

Reply:
But labeling when spending was removed from `spend_utxos_to_output`? Meanwhile labeling when spending was added to `sweep`. None of that seems to rely on any of the fee estimation calls. Just seems a bit inconsistent and unrelated.

Reply:
My goal was to make it so calls that just make a PSBT don't label, and the calls that create and broadcast the transaction do. So for `spend_utxos_to_output` the labeling happens in `event.rs` because that is where the transaction is finalized and broadcast. Labeling is done in `sweep` now because that's where the final transcation is broadcast as well.

Reply:
I see, perhaps the name `spend_utxos_to_output` should be changed then as it does not spend.

Reply:
updated name to `create_sweep_psbt_to_output`